### PR TITLE
Style

### PR DESCRIPTION
**Pull request by Farida-Hamid**
This is the last project for the Bookstore website. Now that all the functionality is done, I added styling. I used Zeplin as a collaboration tool.

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 
 const Header = () => (
-  <header>
+  <header className="font1">
     <h1>Library</h1>
     <nav>
       <Link to="/">BOOKS</Link>
```
Comment (GzimAsani) [inline src/components/Header.js]: - [x] The categories color is different from the design, kindly update it 
YOurs: image
Design: image

File: src/components/Book.js [hunk 2]
```diff
@@ -13,12 +15,11 @@ const Book = ({ title, author, id }) => {
...
-      <h2>{title}</h2>
-      <h3>{author}</h3>
-      id:
-      {id}
-      <button type="button" onClick={deleteBook} value={id}>Delete</button>
+    <div className="book font2">
+      <text className="category font1">{category}</text>
+      <h2 className="title">{title}</h2>
+      <h3 className="author">{author}</h3>
+      <button className="delete" type="button" onClick={deleteBook} value={id}>Delete</button>
```
Comment (GzimAsani) [inline src/components/Book.js]: - [x] The design looks a lot different from yours, kindly updated it according to your design

YOurs: image

Design:image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @Farida-Hamid             :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your styling  :+1:  
 
 ## Required Changes 

- [x] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js [hunk 3]
```diff
@@ -13,12 +15,11 @@ const Book = ({ title, author, id }) => {
...
-      id:
-      {id}
-      <button type="button" onClick={deleteBook} value={id}>Delete</button>
+    <div className="book font2">
+      <text className="category font1">{category}</text>
+      <h2 className="title">{title}</h2>
+      <h3 className="author">{author}</h3>
+      <button className="delete" type="button" onClick={deleteBook} value={id}>Remove</button>
     </div>
   );
```
Comment (fobadara) [inline src/components/Book.js]: [REQUIRED] I that the previous reviewer didn't go into details. But what he means by looks a lot different includes the blue colour of the author's name, the percentage circle and every other thing in the Zeplin design irrespective of whether you created the functionality for them or not

| Zeplin Template|
|---------------|
|image|

File: src/components/Header.js [hunk 4]
```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import login from './img/login.png';
 
 const Header = () => (
-  <header>
-    <h1>Library</h1>
-    <nav>
-      <Link to="/">BOOKS</Link>
-      <Link to="/categories">CATEGORIES</Link>
```
Comment (fobadara) [inline src/components/Header.js:9]: [REQUIRED] Kindly make the categories text even lighter

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRES CHANGES ♻️ 
Hi @Farida-Hamid, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
✅ Linters are passing
✅ Good job following the best practices for commit messages

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
[OPTIONAL] I think you should tell us how we can get your code to our local machine and use it in your README
[OPTIONAL] The link to your MIT license in your README returns a 404 error

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @Farida-Hamid,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- No linter errors ✔️
- Professional readme file ✔️

image

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**